frontend/vuejs/src/views/Subjects/AddCreateSubject.vue

```diff
@@ -165,18 +168,49 @@ export default {
     this.getData();
     if (this.id) {
       this.getSubject();
+      this.listCourse();
     }
   },
 
   methods: {
-    getData() {
-      this.$store.dispatch("subject/getSubject").then((res) => {
-        this.courseData = res.Course;
+    async getData() {
+      await this.$store.dispatch("subject/course").then((res) => {
+        this.courseData = res.data;
       });
     },
-    getSubject() {
-      this.$store.dispatch("subject/getDataById", this.id).then((response) => {
-        this.Subject = response;
+    async getSubject() {
+      await this.$store
+        .dispatch("subject/getSubjectById", this.id)
+        .then((response) => {
+          this.Subject.name = response.data.name;
+          this.Subject.description = response.data.description;
+          this.Subject.is_active = response.data.is_active;
+        });
+    },
+    async update() {
+      for (let index = 0; index < this.value.length; index++) {
+        this.Subject.course_id[index] = this.value[index].id;
+      }
+      if (document.getElementById("an").checked) {
+        this.Subject.is_active = false;
+      } else this.Subject.is_active = true;
```

Comment:
document.getElementById("an").checked trả về true/false rồi 
em chỉ cần viết 
this.Subject.is_active = document.getElementById("an").checked;